test/CDPosition.js

```diff
@@ -66,6 +68,34 @@ describe("CDPosition test suit", async function () {
         });
     });
 
+    describe("Add and remove shares from position", () => {
+        beforeEach(async function () {
+            // create a new position
+            await cdp.createPosition(NFT_ID, BASIC_OUSD_PRINCIPLE);
+        });
+
+        it("Should add shares to position", async function () {
+            // validate CDP Values
+            await validateCDP(NFT_ID, BASIC_OUSD_PRINCIPLE, 0, BASIC_OUSD_PRINCIPLE, 0, 0);
+            // create a new position
+            await cdp.addSharesToPosition(NFT_ID, 20);
+            await validateCDP(NFT_ID, BASIC_OUSD_PRINCIPLE, 0, BASIC_OUSD_PRINCIPLE, 0, 20);
+        });
+
+        it("Should remove shares from position", async function () {
+            // create a new position
+            await cdp.addSharesToPosition(NFT_ID, 20);
```

Comment:
Can still pass expect even if both of those fail. Maybe remove a different number of shares and expect the diff

Reply:
fixed!